src/aws-cpp-sdk-core/source/http/curl/CurlHttpClient.cpp

```diff
@@ -699,17 +699,13 @@ std::shared_ptr<HttpResponse> CurlHttpClient::MakeRequest(const std::shared_ptr<
             curl_easy_setopt(connectionHandle, CURLOPT_SSL_VERIFYPEER, 1L);
             curl_easy_setopt(connectionHandle, CURLOPT_SSL_VERIFYHOST, 2L);
 
-#if LIBCURL_VERSION_MAJOR >= 7
-#if LIBCURL_VERSION_MINOR >= 34
-#if defined(ENFORCE_TLS_V1_3)
+#if defined(ENFORCE_TLS_V1_3) && LIBCURL_VERSION_NUM >= 0x073400 // 7.52.0
             curl_easy_setopt(connectionHandle, CURLOPT_SSLVERSION, CURL_SSLVERSION_TLSv1_3);
-#elif defined(ENFORCE_TLS_V1_2)
+#elif defined(ENFORCE_TLS_V1_2) && LIBCURL_VERSION_NUM >= 0x072000 // 7.34.0
```

Comment:
0x072000 is 7.32.0
0x072200 is 7.34.0

Documentation lists CURL_SSLVERSION_TLS_v1 as available from 7.34.0 as well, although I see it in curl.h as far back as 7.9.2 which is very old so that #else is fine

Reply:
> 0x072000 is 7.32.0
> 0x072200 is 7.34.0

copy paste error fixing really quickly

> CURL_SSLVERSION_TLS_v1

From this [documentation](https://curl.se/libcurl/c/CURLOPT_SSLVERSION.html) I see `CURL_SSLVERSION_TLSv1_1` being listed as 7.34.0 where as `CURL_SSLVERSION_TLS_v1` has no version

Reply:
You are right, I was looking at CURL_SSLVERSION_TLSv1_0
I'm ready for the weekend :)

Reply:
me too! let me merge this for you really quickly first